packages/docs/docs/api/javascript/defineConsts.mdx

```diff
@@ -0,0 +1,69 @@
+---
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+sidebar_position: 3
+---
+
+# `stylex.defineConsts`
+
+Creates global style constants that can be imported and used within `stylex.create` calls anywhere in your codebase. This is useful for static, string-based style values—especially media queries, which can’t be expressed as CSS variables.
+
+```ts
+function defineConsts<Consts extends { [key: string]: string }>(
+  consts: Consts,
+): Consts<{ [key in keyof Consts]: string }>;
+```
+
+### Example use
+
+You must define your constants as named exports in a `.stylex.js` (or `.stylex.ts`) file:
```

Comment:
Presumably you can mix defineConsts and defineVars in these files? Maybe we need to be really clear that consts get inclined whereas vars turn into CSS variables